test/python/quantum_info/operators/symplectic/test_sparse_pauli_op.py

```diff
@@ -140,6 +140,15 @@ def test_sparse_pauli_op_init(self):
             coeffs[:] = 0
             self.assertEqual(spp_op, ref_op)
 
+    def test_sparse_pauli_op_init_long_ys(self):
+        """Test heavy-weight SparsePauliOp initialization."""
+        with self.subTest(msg="make SparsePauliOp from string of Y's"):
+            yyy = SparsePauliOp("Y" * 100)
+            print(yyy.coeffs)
+            yy = SparsePauliOp("Y" * 99)
+            print(yy.coeffs)
+            self.assertEqual(yyy.coeffs, yy.coeffs)
```

Comment:
We don't need a `subTest` since there's only a single assertion statement in the function, also I think it would be better to use a fixed comparison value -- theoretically both 99 and 100 Y terms could be equal but wrong 🙂 We could e.g. just do
```suggestion
        y = SparsePauliOp("Y" * 1000) 
        self.assertEqual(1, y.coeffs[0])
```

Reply:
addressed

Reply:
not exactly, you still check that both values are equal, but not that they equal 1 as suggested above

Reply:
you're right im sorry i missed this completely!, I just processed the subTest isssue.